README.md

```diff
@@ -1,7 +1,33 @@
-# RPC Beta
+# Remote Provisioning Client (RPC) Beta
 
 > Disclaimer: Production viable releases are tagged and listed under 'Releases'. All other check-ins should be considered 'in-development' and should not be used in production
 
+Used for activation, deactivation, maintenance, and status of AMT
```

Comment:
This is good info, but we keep this kind of info out of the readme and keep it in the docs. Can you move it to https://open-amt-cloud-toolkit.github.io/docs/2.5/Reference/RPC/commandsRPC/#maintenance, or ask @bwendlandt-intel  for help and he can do it.